### DISCUSSION TITLE
Making instructions

### DISCUSSION THREAD

**Discussion by edsousa84** [leozide/leocad#776] [Q&A] [answered]
Hello I am tring to make some instructions for a lego model I made but I am having trouble generating them.
I am on a ubuntu 18.04 machine.

I configure them at File -> Instructions
I change the background and make it show the steps number and part list
But when I print it to PDF it does  show the Step number but it does not show the parts list or changes the backgroud color.

What am I doing wrong?

**Comment by leozide** [comment, marked as answer]
The Instructions Dialog is not fully implemented yet, it was enabled by mistake in the builds.

**Reply by edsousa84** [reply]
Oh ok is there anightly build I can use that has more implemented (even if it has some bugs)?

**Reply by nathaneltitane** [reply]
you can build the git version.

**Reply by edsousa84** [reply]
Is there a eta for this feature?

**Reply by nathaneltitane** [reply]
it's been 'in progress' for a couple of years now... asking for an ETA is irrelevant lol

**Reply by leozide** [reply]
There's a 'Print' option in the instructions dialog, you can try the continuous release here: https://github.com/leozide/leocad/releases/tag/continuous

**Reply by edsousa84** [reply]
OK thank you.
